package.json

```diff
@@ -617,21 +617,30 @@
     ]
   },
   "scripts": {
-    "vscode:prepublish": "tsc -p ./",
-    "compile": "tsc -watch -p ./",
+    "vscode:prepublish": "rimraf dist && webpack --mode production",
```

Comment:
seems dist is not removed in edge extension? need add it also?

Reply:
Not really required for Edge because there is only one JS file in dist which will be overwritten every time. For toolkit, in dist there is a script folder and a fallbacks folder container clipboardy binaries.